examples/mongo/README.md

```diff
@@ -0,0 +1,25 @@
+# Mongo Metrics
+
+The [`mongo_3.rb`](mongo_3.rb) script generates metrics from the `mongotop` and `mongostat` commands for a Mongo 3.0 database. The following metrics will be output for every collection in your system:
```

Comment:
Cool.  Next PR then, and if anyone asks, "mongo 3 has better tools that made it easy" is a pretty reasonable answer.